splat.yaml

```diff
@@ -25,7 +25,8 @@ segments:
     - [0x1000, hasm, entry]
     - [0x1060, c, sys/main]
     - [0x1510, hasm, padding_1510]
-    - [0x1520, c]
+    - [0x1520, c, sys/sched]
+    - [0x35E0, c, 1520]
```

Comment:
probably best to rename this file to 35E0 to avoid confusion

Reply:
done